docs/_data/home/benefits.yml

```diff
@@ -0,0 +1,11 @@
+- heading: Lower your messaging costs
+  description: |
+    Pay less for messages with out competitive rates and never pay any service or integration fees.
+
+- heading: Increase your operating efficiency
+  description: |
+    Make procuring messages easier by using NHS Notify as a centralised all in one provider.
```

Comment:
"all-in-one" rather than "all in one"

Reply:
The Figma prototype it's "all in one"

Reply:
We are still a quality gate for the design. This is an example of a "multi-word adjective before a noun", (as is "multi-word" as the word "adjective" in that context is a noun)!

Reply:
Mark is correct here, good spot!

Reply:
Updated to be all-in-one